WebMVC/Views/Catalog/CreateEventPost.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@using WebMVC.Services
+@using WebMVC.ViewModels
+@{
+    ViewData["Title"] = "Create Event";
+}
+
+<h2>NiceJob!! Event Created Successfully!!</h2>
+<form asp-controller="Catalog" asp-action="Index" method="get">
+    <input type="submit" value="Go Back to Home Page"/>
```

Comment:
We dont need GoBackToHomePage button, as we always can go back to home page by clicking on the logo.

Reply:
I thought, it would be better to give one more option. We can change the text as Continue Browsing Event.

Reply:
ok